src/core/atomicdex/pages/qt.settings.page.cpp

```diff
@@ -606,12 +585,84 @@ namespace atomic_dex
             }
         };
 
-        functor_remove(std::move(wallet_cfg_path));
+        if (fs::exists(wallet_cfg_path))
+        {
+            nlohmann::json wallet_config_json_data;
+            nlohmann::json active_list = nlohmann::json::array();
```

Comment:
I think this container is suboptimal for what you are trying to achieve.

Take a look at <https://en.cppreference.com/w/cpp/container/unordered_set> - an `unordered_set<std::string>` seems more appropriate and the following usage <https://en.cppreference.com/w/cpp/container/unordered_set/contains>.